site/content/news/rdf4j-360.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "RDF4J 3.6.0 released"
+date: 2021-02-22T10:00:00+0100
+layout: "single"
+categories: ["news"]
+---
+We are pleased to announce that RDF4J 3.6.0 is now available. This is a minor release with about 19 
+improvements and bug fixes.
+
+For full details, have a look at the [release notes](/release-notes/3.6.0). Below, we briefly highlight some
+of the major improvements.
+
+<!--more-->
+
+## Improved SHACL Implementation
+RDF4J likely sports the worlds only SHACL implementation capable of analyzing your transactional changes.
+As you scale your database to hundreds of millions of triples you can trust our ShaclSail to take care of 
+your data integrity with no more overhead than when you started out with just a few thousand triples. 
+
+With RDF4J 3.6.0 our SHACL implementation is inching just that bit closer to full spec compliance with 
+support for `sh:node` and `sh:property` to their fullest expressiveness. You can now nest `sh:property`
+and `sh:node` to build richer shapes than ever before. 
+
+This release also contains several bug fixes to the ShaclSail making it the most stable and reliable validation
+engine we have ever shipped.
+
+
+## Faster Isomorphism Detection
+By implementing Aidan Hogan's algorithm for faster graph isomorphism detection`Models.isomorphic(...)`
+has been radically improved. Where the previous algorithm could take minutes to compare two models, the
+new implementation only takes seconds. Even models that previously were impossible to compare due to 
+their size can now quickly and efficiently be compared.
+
+Comparing two models using `Models.isomorphic(...)` allows you to compare models that contain blank 
+nodes. Blank nodes are mapped to each other based on their contents instead of their identifier. In the
+RDF4J codebase we regularly use this in our tests to compare a resulting model to an expected result. 
+
+Hogan's algorithm has been shown to have very good performance characteristics for almost all practical
+cases, and is guaranteed to terminate in reasonable time. Read the full paper here: [Hogan, A., 2017. Canonical forms for isomorphic and equivalent RDF graphs: algorithms for leaning and labelling blank nodes. ACM Transactions on the Web (TWEB), 11(4), pp.1-62.](http://aidanhogan.com/docs/rdf-canonicalisation.pdf)
```

Comment:
I've rewritten your copy on the new isomorphism algorithm. What do you think @jeenbroekstra ?

Reply:
Very nice!